frontend/src/views/admin/InviteCodePanelView.vue

```diff
@@ -0,0 +1,270 @@
+<template>
+  <div class="flex w-full justify-center">
+    <alert-box
+      @close="pageNotification = ''"
+      v-if="pageNotification"
+      :scheme="alertSchemes.success"
+    >
+      {{ pageNotification }}
+    </alert-box>
+    <alert-box
+      @close="pageError = ''"
+      v-if="pageError"
+    >
+      {{ pageError }}
+    </alert-box>
+  </div>
+  <admin-nav>
+
+  </admin-nav>
```

Comment:
Is there supposed to be a child element here or can we use a stand-alone tag?

Reply:
Reformatting / editor shenanigans, I'll clean that up!